pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -711,23 +711,23 @@ func TestPodContainerDeviceAllocation(t *testing.T) {
 			expectedContainerOptsLen:  []int{3, 2, 2},
 			expectedAllocatedResName1: 2,
 			expectedAllocatedResName2: 1,
-			expErr:                    nil,
+			expErr: nil,
```

Comment:
I believe Gofmt is going to complain about this :) In go, the expectation is that the values for all maps are aligned.